src/data/navigation/sections/reference.js

```diff
@@ -0,0 +1,83 @@
+module.exports = [
+    {
+        title: 'GraphQL Mesh',
+        path: '/reference/index.md'
+    },
+    {
+        title: 'Handlers',
+        path: '/reference/handlers/index.md',
+        header: true,
+        pages: [
+            {
+                title: 'OpenAPI',
+                path: '/reference/handlers/openapi.md'
+            },
+            {
+                title: 'GraphQL',
+                path: '/reference/handlers/graphql.md'
+            },
+            {
+                title: 'JSON Schemas',
+                path: '/reference/handlers/json-schema.md'
+            },
+        ]
+    },
+    {
+        title: 'Transforms',
+        path: '/reference/transforms/index.md',
+        header: true,
+        pages: [
+            {
+                title: 'Transforms Overview',
+                path: '/reference/transforms/transforms-introduction.md'
+            },
+            {
+                title: 'Encapsulate',
```

Comment:
Change all these titles to their programmatic names: encapsulate, filterSchema, etc.